package.json

```diff
@@ -32,6 +31,9 @@
     "browser",
     "storage"
   ],
+  "engines": {
+    "node": ">=12.0.0"
+  },
```

Comment:
is this really needed?

Reply:
I'm trying to remember why I added this. I re-ran the tests with the engine removed, passes all tests. So I'll remove accordingly.